src/Visualize.jl

```diff
@@ -28,4 +28,6 @@ function contour2D_on_globe! end
 
 function heatmap2D_on_globe! end
 
+function plot_bias! end
```

Comment:
```suggestion
function plot_bias_on_globe! end
```
I think we should be consistent with the other names in this function